server/routes/support/handlers/offenderDetail.ts

```diff
@@ -73,6 +104,16 @@ export default class OffenderDetailRoutes {
         pdu: probationPractitioner?.team?.borough?.description,
         region: probationPractitioner?.probationArea?.description,
       },
+      license: {
```

Comment:
This should be `licence`, which will affect all other references to the object

Reply:
Done.

Reply:
It still appears to be called `license`